zulip/integrations/bridge_with_slack/README.md

```diff
@@ -5,30 +5,42 @@ This is a bridge between Slack and Zulip.
 ## Usage
 
 ### 1. Zulip endpoint
```

Comment:
This likely needs to document this requires Zulip 10+, right?

Reply:
Yes. I added this in the documentation header:
```markdown
# Slack <--> Zulip bridge

This is a bridge between Slack and Zulip.

!!! warn ""

    This setup requires at least Zulip version 10.0.

## Usage
```